modules/ROOT/pages/apikit-graphql-api-implementation.adoc

```diff
@@ -0,0 +1,505 @@
+
+
+==== Get a Response
+
+{
+   "data": {
+       "postComment": {
+           "id": "201",
+           "body": "Nice post! Thanks for sharing! It's just what I was looking for",
+           "post": {
+               "id": "101",
+               "title": "GraphQL by example",
+               "comments": [
+                   {
+                       "id": "201",
+                       "user": {
+                           "username": "ekosex"
+                       },
+                       "body": "Nice post! Thanks for sharing! It's just what I was looking for"
+                   }
+               ]
+           }
+       }
+   }
+}
+
+
+=== Reply to the New Comment in the Post
+
+This example request adds a comment of the user that created the original post in response to the previous comment. The query asks the service to return all the information of the post in addition to the new reply.
+
+==== Make a Query
+
+mutation ReplyToComment($commentId: ID!, $userId: ID!, $body: String!) {
+   replyToComment(commentId: $commentId, userId: $userId, body: $body) {
+       id
+       body
+       post {
+           id
+           title
+           comments {
+               id
+               body
+               replies {
+                   id
+                   user {
+                       username
+                   }
+                   body
+               }
+           }
+       }  
+   }
+}
+
+
+==== Make Variables
+
+{
+   "commentId": 201,
+   "userId": 42,
+   "body": "Thank you!"
+}
+
+
+==== Trigger a Request
+
+curl --location --request POST 'http://localhost:8081/graphql' \
+--header 'Content-Type: application/json' \
+--data-raw '{"query":"mutation ReplyToComment($commentId:ID!,$userId:ID!,$body:String!){replyToComment(commentId:$commentId,userId:$userId,body:$body){id body post{id title comments{id body replies{id user{username}body}}}}}","variables":{"commentId":201,"userId":42,"body":"Thank you!"}}'
+
+
+==== Get a Response
+
+{
+   "data": {
+       "replyToComment": {
+           "id": "202",
+           "body": "Thank you!",
+           "post": {
+               "id": "101",
+               "title": "GraphQL by example",
+               "comments": [
+                   {
+                       "id": "201",
+                       "body": "Nice post! Thanks for sharing! It's just what I was looking for",
+                       "replies": [
+                           {
+                               "id": "202",
+                               "user": {
+                                   "username": "tblackmore15"
+                               },
+                               "body": "Thank you!"
+                           }
+                       ]
+                   }
+               ]
+           }
+       }
+   }
+}
+
+
+
+
+== See Also
+
+* xref:doc.adoc[Some Relevant Doc Article]
```

Comment:
Remove this

Reply:
I'm still thinking about what link could be included, and if it should be included. So I'll leave this comment as a reminder ;)
Thanks!